Tests/Unit/Controller/TeaControllerTest.php

```diff
@@ -51,6 +65,16 @@ protected function setUp(): void
         /** @var TeaRepository&ProphecySubjectInterface $teaRepository */
         $teaRepository = $this->teaRepositoryProphecy->reveal();
         $this->subject->injectTeaRepository($teaRepository);
+
+        $this->responseFactoryProphecy = $this->prophesize(ResponseFactory::class);
+        /** @var ResponseFactory&ProphecySubjectInterface $responseFactory */
+        $responseFactory = $this->responseFactoryProphecy->reveal();
+        $this->subject->injectResponseFactory($responseFactory);
+
+        $this->streamFactoryProphecy = $this->prophesize(StreamFactory::class);
+        /** @var StreamFactory&ProphecySubjectInterface $streamFactory */
+        $streamFactory = $this->streamFactoryProphecy->reveal();
+        $this->subject->injectStreamFactory($streamFactory);
```

Comment:
You can simplify this a lot by adding `htmlResponse` to the list of mocked methods in line 58.

Reply:
Done